packages/ez_search/ez_search.0.2.1/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+synopsis: "A set of two complementary packages for fulltext searches"
+description: """\
+ocp-search is a tool to grep a pre-indexed set of sources.
+ez_search is a library to search a pre-indexed set of files.
+"""
+authors: ["Fabrice Le Fessant <fabrice.le_fessant@origin-labs.com>"]
+maintainer: ["Fabrice Le Fessant <fabrice.le_fessant@origin-labs.com>"]
+homepage: "https://ocamlpro.github.io/ez_search"
+doc: "https://ocamlpro.github.io/ez_search/sphinx"
+bug-reports: "https://github.com/ocamlpro/ez_search/issues"
+dev-repo: "git+https://github.com/ocamlpro/ez_search.git"
+tags: "org:ocamlpro"
+build: [
+  ["dune" "subst"] {dev}
+  ["sh" "-c" "./scripts/before.sh build '%{name}%'"]
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+  ["sh" "-c" "./scripts/after.sh build '%{name}%'"]
+]
+install: [
+  ["sh" "-c" "./scripts/before.sh install '%{name}%'"]
+]
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "dune" {>= "2.7.0"}
+  "re" {>= "1.8.0" & < "2.0.0"}
+  "parmap" {>= "1.2.3" & < "2.0.0"}
+  "ez_file" {>= "0.2"}
+  "ez_cmdliner" {>= "0.2"}
+  "ppx_inline_test" {with-test}
+  "ppx_expect" {with-test}
+  "odoc" {with-doc}
+  "ocamlformat" {with-test}
+]
+conflicts: [ "base-domains" ]
+available: os = "linux" & arch != "arm32" & arch != "arm64" & arch != "ppc64" & arch != "s390x"
```

Comment:
```suggestion
available: os = "linux" & arch = "x86_64"
```

Reply:
@mseri, it doesn't work on macOS ?

Reply:
No idea. I did not put there the first constraint, `os = “linux”` was already there. But it does not compile on architectures different from x86_64

Reply:
OK then I believe we could remove "linux" and we will constraint it if it doesn't work elsewhere...